src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java

```diff
@@ -78,6 +78,9 @@ public class Errand {
 	@Schema(description = "Id for the group which is currently assigned to the errand if a group is assigned", example = "hardware support")
 	private String assignedGroupId;
 
+	@Schema(description = "Email address used for escalation of errand", example = "joe.doe@email.com")
```

Comment:
Should the email address be validated? 
Maybe add this?
`@Email`

Reply:
If validation is added, some more tests will be needed as well.

Reply:
added